core/src/utils/test/playwright/viewports/index.ts

```diff
@@ -0,0 +1,4 @@
+export const TabletViewport = {
```

Comment:
This is probably fine for now, but we may want to consider making `Viewports` a constant and differentiating between portrait/landscape for the viewport values; if we find ourselves needing to capture screenshots on common viewports.

i.e.:
```ts
export cont Viewport = {
  tablet: {
    portrait: {
      width: 768,
      height: 900
    }
  }
}
```

Reply:
Good thought. I made that change.